templates/header.php

```diff
@@ -1,7 +1,7 @@
-<header class="banner" role="banner">
+<header class="banner">
   <div class="container">
     <a class="brand" href="<?= esc_url(home_url('/')); ?>"><?php bloginfo('name'); ?></a>
-    <nav role="navigation">
+    <nav class="site-nav">
```

Comment:
can you make this `nav-primary` please?

i'd like to roll with this since the `<ul>` class is `nav`, and primary is the name of the navigation menu